src/ert/cli/model_factory.py

```diff
@@ -129,7 +135,37 @@ def _setup_ensemble_experiment(
             minimum_required_realizations=config.analysis_config.minimum_required_realizations,
             ensemble_size=config.model_config.num_realizations,
             stop_long_running=config.analysis_config.stop_long_running,
-            experiment_name=args.experiment_name,
+            experiment_name=experiment_name,
+        ),
+        config,
+        storage,
+        config.queue_config,
+    )
+
+
+def _setup_evaluate_parameters(
+    config: ErtConfig, storage: StorageAccessor, args: Namespace
+) -> EvaluateParameters:
+    min_realizations_count = config.analysis_config.minimum_required_realizations
+    active_realizations = _realizations(args, config.model_config.num_realizations)
+    active_realizations_count = int(np.sum(active_realizations))
```

Comment:
Since `_realizations` returns array of `bool`, the conversion to `int` should be unnecessary.

```suggestion
    active_realizations_count = np.sum(active_realizations)
```